Core/src/org/sleuthkit/autopsy/discovery/SizeFilterPanel.java

```diff
@@ -0,0 +1,187 @@
+        sizeList.setVisibleRowCount(5);
+        sizeScrollPane.setViewportView(sizeList);
+
+        javax.swing.GroupLayout layout = new javax.swing.GroupLayout(this);
+        this.setLayout(layout);
+        layout.setHorizontalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addComponent(sizeScrollPane, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(layout.createSequentialGroup()
+                .addComponent(sizeScrollPane, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                .addGap(0, 0, 0))
+        );
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void sizeCheckboxActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_sizeCheckboxActionPerformed
+        sizeList.setEnabled(sizeCheckbox.isSelected());
+    }//GEN-LAST:event_sizeCheckboxActionPerformed
+
+
+    // Variables declaration - do not modify//GEN-BEGIN:variables
+    private javax.swing.JCheckBox sizeCheckbox;
+    private javax.swing.JList<FileSize> sizeList;
+    private javax.swing.JScrollPane sizeScrollPane;
+    // End of variables declaration//GEN-END:variables
+
+    @Override
+    void configurePanel(boolean selected, int[] indicesSelected) {
+        sizeCheckbox.setSelected(selected);
+        if (sizeCheckbox.isEnabled() && sizeCheckbox.isSelected()) {
+            sizeScrollPane.setEnabled(true);
+            sizeList.setEnabled(true);
+            if (indicesSelected != null) {
+                sizeList.setSelectedIndices(indicesSelected);
+            }
+        } else {
+            sizeScrollPane.setEnabled(false);
+            sizeList.setEnabled(false);
+        }
+    }
+
+    @Override
+    JCheckBox getCheckbox() {
+        return sizeCheckbox;
+    }
+
+    @Override
+    JLabel getAdditionalLabel() {
+        return null;
+    }
+
+    /**
+     * Initialize the file size filter.
+     */
+    private void setUpSizeFilter(FileSearchData.FileType fileType) {
+        int count = 0;
+        DefaultListModel<FileSize> sizeListModel = (DefaultListModel<FileSize>) sizeList.getModel();
+        sizeListModel.removeAllElements();
+        if (null == fileType) {
+            for (FileSize size : FileSize.values()) {
+                sizeListModel.add(count, size);
+            }
+        } else {
+            List<FileSearchData.FileSize> sizes;
+            switch (fileType) {
+                case VIDEO:
+                    sizes = FileSize.getOptionsForVideos();
+                    break;
+                case IMAGE:
+                    sizes = FileSize.getOptionsForImages();
+                    break;
+                case DOCUMENTS:
+                    sizes = FileSize.getOptionsForImages();
```

Comment:
Can we maybe rename the image sizes to be more "default size options" (in various places) so it won't seem weird that we're using it for images and videos?

Reply:
yeah good call